openhcl/underhill_core/src/dispatch/mod.rs

```diff
@@ -48,6 +49,7 @@ use socket2::Socket;
 use state_unit::SavedStateUnit;
 use state_unit::SpawnedUnit;
 use state_unit::StateUnits;
+use std::future;
```

Comment:
no need to do this sort of explicit import - just specifying `std::future::pending::<()>().await` below should be a-okay